pkgs/by-name/di/dita-ot/package.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, fetchzip
+, bash
+, jdk17
+, nix
+, coreutils
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  name = "dita-ot";
+  version = "4.2.1";
+
+  buildInputs = [
+    bash
+    jdk17
+  ];
+
+  src = fetchzip {
+    url = "https://github.com/dita-ot/dita-ot/releases/download/${version}/dita-ot-${version}.zip";
+    hash = "sha256-PtscKsQwW1uBu0OfVHi7/d9JujGx12nZT+W3uMPQQWc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/
+    cp -r $src/* $out/
+
+    runHook postInstall
+  '';
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+    $out/bin/dita --version
+    runHook postInstallCheck
+  '';
+
+  meta = {
+    homepage = "https://dita-ot.org";
+    description = "The open-source publishing engine for content authored in the Darwin Information Typing Architecture.";
+    license = lib.licenses.asl20;
+    mainProgram = "dita";
+    platforms = lib.platforms.all;
```

Comment:
Given that this package is not built from source, there should be a [source provenance](https://nixos.org/manual/nixpkgs/stable/#sec-meta-sourceProvenance) attribute.

Reply:
Implemented 👍🏼